engines/director/lingo/tests/getNthFileNameInFolder.lingo

```diff
@@ -0,0 +1,2 @@
+set n = getNthFileNameInFolder("@", 1)
```

Comment:
I would like to see a test that will check that a file is being mixed. E.g. since you're running `getNthFileNameTestFolder.lingo` file, it should be in path as well, so please mix the file into the directory where it is found. I think,it will be also "root", which is represented as "@".

Also, instead of creating a new file, please put this test into `buitlin.lingo`